### ISSUE TITLE
no blueprint named default exists

### ISSUE THREAD

**Issue by amalshaji** [charludo/barely#9]
**Describe the bug**
Error while running `barely new`

**To Reproduce**
```bash
(.venv) ➜  barely-test pip install barely
(.venv) ➜  barely-test barely        
[barely][  core][ERROR] :: could not find 'config.yaml'. Exiting
(.venv) ➜  barely-test barely new
[barely][  core][ERROR] :: no blueprint named default exists.
```

**Expected behavior**
```bash
$ barely new
[barely][  core][ INFO] :: setting up new project with parameters:
                        ->   webroot: webroot
                        ->   devroot: devroot
                        -> blueprint: default
[barely][  core][ INFO] :: setting up basic config...
[barely][  core][ INFO] :: done.
```

**Desktop (please complete the following information):**
```bash
➜  lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal
```
WSL2

**Comment by charludo** [comment]
Huh, that's weird! Could you check `barely blueprints` to see if any are found at all?

**Comment by amalshaji** [comment]
```bash
[barely][  core][ INFO] :: found 0 blueprints:
```

**Comment by charludo** [comment]
OK. Does `barely --debug new` output anything remotely sensible as the expected blueprint locations? If so, is there anything there?

**Comment by amalshaji** [comment]
```bash
(.venv) ➜  barely-test barely --debug new
[barely][  core][DEBUG] :: Logging setup complete
[barely][  core][DEBUG] :: platform: Unix-like
[barely][  core][DEBUG] :: checked / created plugins/content
[barely][  core][DEBUG] :: checked / created plugins/backup
[barely][  core][DEBUG] :: checked / created plugins/publication
[barely][  core][DEBUG] :: checked / created blueprints
[barely][  core][DEBUG] :: created all required dirs
[barely][  core][DEBUG] :: platform: Unix-like
[barely][  core][DEBUG] :: looking for blueprints in: /home/amalshaji/Workspace/barely-test/.venv/lib/python3.9/site-packages/blueprints
[barely][  core][DEBUG] :: looking for blueprints in: /home/amalshaji/.barely/blueprints
[barely][  core][ERROR] :: no blueprint named default exists.
(.venv) ➜  barely-test ls
(.venv) ➜  barely-test ls ~/.barely 
blueprints  plugins
(.venv) ➜  barely-test ls ~/.barely/blueprints
```

The folder is empty

**Comment by amalshaji** [comment]
it works

**Comment by charludo** [comment]
Should be fixed with [b68001b](https://github.com/charludo/barely/commit/b68001b14496b0964b0924524630ea092d38675c).   The error originated from the way packages are bundled for PyPI, the blueprints were simply not included in that. Moved the entire `blueprints` folder into `barely`-proper.

**Comment by charludo** [comment]
Fix is live on [PyPI](https://pypi.org/project/barely/1.0.3/), so I'll close the issue. 

Thank you for the bug report!
